src/main/webapp/college-info.html

```diff
@@ -15,10 +15,9 @@
     <main>
       <nav class="topnav">
         <a class="active" href="college-info.html">School Info</a>
-        <a id="reviews-button">Student Reviews</a>
+        <a href="comments.html">Student Reviews</a>
```

Comment:
I am going to let you merge first and then I will merge after because I changed the nav bar